xc/xenops_server_xen.ml

```diff
@@ -2440,7 +2442,7 @@ module Actions = struct
 		device_watches := IntMap.add domid [] !device_watches
 
 	let domain_disappeared xc xs domid =
-		let token = Xenstore_watch.token_of_domain domid in
+		let token = Printf.sprintf "xenopsd-xc:domain-%d" domid in
```

Comment:
Why not update `Xenstore_watch.token_of_domain` to return the new string? Now we have the same hardcoded string in many places.